tests/common_test.h

```diff
@@ -43,16 +43,14 @@
 
 #include <json-c/json.h>
 #ifndef BUILD_ASE
-#include <log_int.h>
+#include "common_int.h"
```

Comment:
Do we still need this since these tests will only be building for ASE?

Reply:
gtAse was built with these test cases.